dstep/translator/Context.d

```diff
@@ -33,8 +34,12 @@ class Context
     private TypedefIndex typedefIndex_ = null;
     private Translator translator_ = null;
     private Output globalScope_ = null;
+    private Cursor[string] typeNames_;
 
-    public this(TranslationUnit translUnit, Options options, Translator translator)
+    public MacroDefinition[string] macroDefinitions;
+    string macroLinkage = "extern (D)";
```

Comment:
Currently I cannot come up with a use case for that. Perhaps keep it simple and not make that possible.